spec/decorators/project_decorator_spec.rb

```diff
@@ -287,5 +287,73 @@
     end
 
   end
+
+  describe "#edit_field_available?" do
```

Comment:
eu mudaria esse nome pra `editable` ou algo do tipo, daí seus testes ficariam:

`it { expect(project.decorate).not_to be_editable`

💅